PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azuretools/ijidea/actions/AzureSignInAction.java

```diff
@@ -134,23 +136,27 @@ public static void onAzureSignIn(Project project) {
                 });
             }
         } else {
-            doSignIn(authMethodManager, project);
+            doSignIn(authMethodManager, project).subscribe();
         }
```

Comment:
how about moving subscribe() into soSignIn(...) ?

Reply:
no, some other callings need to do something in `subscribe`